update(helm): allow setting different base dir(and others) for cspc and cvc operators (#404)
openebs/cstor-operators: deploy/helm/charts/README.md

| cspcOperator.resyncInterval | string | `"30"` | CSPC operator resync interval |
| cspcOperator.securityContext | object | `{}` | CSPC operator security context |
| cspcOperator.tolerations | list | `[]` | CSPC operator pod tolerations |

> | cspcOperator.baseDir | string | `"/var/openebs"` | CSPC operator base directory for openebs on host path |

Reviewer: Configuration is fed via CSPCOperator but it will propagate to CSPI pods(cStor pool pods), So good to have description something like 
```suggestion
| cspcOperator.baseDir | string | `"/var/openebs"` |  base directory for openebs cStor pools on host path to store pool related information |
```

> | cspcOperator.cstorPoolSparseDir | string | `"/var/openebs/sparse"` | cStor pool sparse directory for sparse devices |

Reviewer: How about below suggestion?
```suggestion
| cspcOperator.SparseDir | string | `"/var/openebs/sparse"` | sparse directory to access sparse based devices |
```
**naming convention**: 
- Since this SparseDir must be created by ndm/manually for cStor to use sparse files so we can avoid the cstorPool in prefix

> | cspcOperator.SparseDir | string | `"/var/openebs/sparse"` | sparse directory to access sparse based devices |

Reviewer: Let's follow existing naming convention i.e cammelCase (Sorry, In my previous comments I should updated to `sparseDir` instead of `SpaseDir`)
```suggestion
| cspcOperator.sparseDir | string | `"/var/openebs/sparse"` | sparse directory to access sparse based devices |
```

| cvcOperator.volumeMgmt.image.registry | string | `nil` | Volume mgmt image registry |
| cvcOperator.volumeMgmt.image.repository | string | `"openebs/cstor-volume-manager"` | Volume mgmt image repository |
| cvcOperator.volumeMgmt.image.tag | string | `"3.0.0"` |  Volume mgmt image tag|
| cvcOperator.baseDir | string | `"/var/openebs"` | CVC operator base directory for openebs on host path |
| cvcOperator.cstorTargetDir | string | `"/var/openebs/sparse"` | cStor target directory for dumping cStor volume |

> | useVarDirectoryPath | bool | `false` | Default hostpath storage for OpenEBS related files |

Reviewer: Can we avoid useVarDirectoryPath variable?
